sessions/js-dom-and-events/demo-end/index.html

```diff
@@ -6,14 +6,13 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
-    <title>JS Variables and Numbers: Social Media Post</title>
+    <title>JS Basics: Demo</title>
```

Comment:
is this not the HTML for the Session JS DOM and Events?

```suggestion
    <title>JS DOM and Events</title>
```